tests/vmaas_sync_tests/test_vmaas_sync.py

```diff
@@ -53,7 +53,8 @@ def check_sync_logs(records, start_offset):
         """Check to see we got the expected msgs from sync_cve_md"""
         assert records[start_offset].msg == 'Syncing CVE metadata'
         assert records[start_offset + 1].msg.startswith('Importing CVE metadata')
-        assert records[start_offset + 2].msg == 'Finished syncing CVE metadata'
+        assert records[start_offset + 2].msg == 'Deleting %s unnecessary CVE metadata'
```

Comment:
Or the CVE-2024-1 could be added to the response of vmaas_mock.py so it won't attempt to delete it. But feel free to keep it like this.